src/components/Landing/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {Link} from "react-router-dom";
+
+export default function Landing() {
+    return (
+        <div class="mt-3 mb-3 h-100">
```

Comment:
@nikhilrayaprolu 
`mt-3` & `mb-3` should be changed to a class that set both of TOP and BOTTOM